rendering_resource_manager_service/session/urls.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env pyth_build_allocation_commandon
```

Comment:
Bloddy mouse wheel button... need to get a new mouse for sure to avoid these random copy/paste... Thanks.